src/Domain/Stager.php

```diff
@@ -38,21 +40,43 @@ public function __construct(Filesystem $filesystem, ProcessFactory $processFacto
     }
 
     /**
-     * @param string[] $command
+     * @param string[] $composerCommand
+     *   The Composer command parts exactly as they would be typed in the
+     *   terminal. Example:
+     *
+     *   @code{.php}
+     *   $command = [
+     *     // "composer" is implied.
+     *     'require',
+     *     'lorem/ipsum:"^1 || ^2"',
+     *     '--with-all-dependencies',
+     *   ];
+     *   @endcode
      *
      * @throws \PhpTuf\ComposerStager\Exception\DirectoryNotFoundException
```

Comment:
Do we need these `@throws`? The actual method doesn't look like it throws any of them.

Reply:
Believe it or not, it can, from deep in the call chain. I'm actually using PHPStan to test `@throws` to tell me both when they're missing _and_ when they're unused. Quite slick, really. It's another instance of where I hate inconsistency due to "judgment calls" each and every time.

Reply:
Yeah, I figured those throws would be coming from deeper in the call stack. I just hate it when we put `@throws` on methods that don't _directly_ throw the exception(s) mentioned -- it means a wild goose chase later when trying to figure out what could ACTUALLY throw a given exception.

I defer to your judgment on this one, but to me this feels like a place where coding standards obfuscate the "truth" of what code might _actually_ do.

Reply:
I feel like the more costly risk is that someone writing calling code wouldn't realize that there are exceptions that they need to catch because they weren't annotated. I'm keeping these.